src/components/ProjectImagesViewer.js

```diff
@@ -19,7 +19,7 @@ function ProjectImagesViewer({ styles, images, isViewerOpen, setIsViewerOpen })
         </div>
         <div className={styles['image-viewer-hero-container']}>
           <img className={styles['image-viewer-hero-image']} src={heroImage.link} alt={heroImage.description}/>
-          <p>{heroImage.description}</p>
+          <p className={styles['image-viewer-hero-discription']}>{heroImage.description}</p>
```

Comment:
nit: spelling:

```suggestion
          <p className={styles['image-viewer-hero-description']}>{heroImage.description}</p>
```

Reply:
Lordy, I looked at that and thought that doesn't look right. Have the correct one right next to it. Sheesh